site2/docs/getting-started-standalone.md

```diff
@@ -23,6 +23,40 @@ Currently, Pulsar is available for 64-bit **macOS**, **Linux**, and **Windows**.
 > **Note**  
 > Broker is only supported on 64-bit JVM.
 
+#### Installing JDK on M1
+In current version, Pulsar uses a BookKeeper version which in turn uses RocksDB which was compiled to work on x86 architecture and not ARM therefor it can only work with x86 JDK. This is planned to be fixed in future versions of Pulsar.
```

Comment:
```suggestion
In current version, Pulsar uses a BookKeeper version which in turn uses RocksDB which was compiled to work on x86 architecture and not ARM therefore it can only work with x86 JDK. This is planned to be fixed in future versions of Pulsar.
```
By saying "the current version", you are referring to 2.10 only, right? If yes, I think this update only applies to 2.10 docs under /site2/website/versioned_docs.

Reply:
@momo-jun No, I mean any version, starting with current one until it is fixed

Reply:
I got your point. I thought RocksDB metadata service was introduced in 2.10 :)
If this change applies to more historical versions of docs, such as 2.10, 2.9 and etc, you can still update them under  /site2/website/versioned_docs.